src/views/ReviewConfirm.vue

```diff
@@ -9,20 +9,20 @@
 
     <section class="mt-10">
       <header>
-        <h2>Review</h2>
+        <h2>Review and Confirm</h2>
+        <p class="mt-1">Review the information in your application. If you need change or complete anything, return to
+          the step make the necessary change.</p>
       </header>
-      <Summary />
+      <Summary class="mt-6"/>
     </section>
-    <section class="mt-10">
+    <section v-if="!isTypeCoop" class="mt-10">
```

Comment:
Maybe we should create a new getter to INCLUDE all types that aren't Coop instead of assuming that everything not a Coop is OK ?

Reply:
Sounds good!

Reply:
Updated! In a few places